bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/pom.xml

```diff
@@ -128,6 +133,14 @@
           <target>${compileSource.1.8}</target>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>${compileSource.1.8}</source>
+          <target>${compileSource.1.8}</target>
+        </configuration>
+      </plugin>
```

Comment:
this looks like a broken merge (its repeated above)

Reply:
done.